src/lib/interp.rs

```diff
@@ -238,22 +257,54 @@ impl VM {
         self.pc = self.bytecode.labels.get(
             &(GLOBAL_NSPACE.to_string(), MAIN_FN.to_string()))
             .expect("Main method not found").clone();
-        self.frames.push(Frame::new(Vec::new(), self.bytecode.bytecode.len()))
+        self.frames.push(Frame::new("main".to_string(), Vec::new(), self.bytecode.bytecode.len()))
+    }
+
+    fn unroll_if_raised(&mut self) {
```

Comment:
Not quite sure I understand the method name.

Reply:
Yes it is a little ambiguous, I've renamed it to `unwind_stack_on_raise`.